tests/Agent/UnitTests/NewRelic.Agent.Extensions.Tests/Lambda/LambdaEventHelpersTests.cs

```diff
@@ -159,11 +204,55 @@ public void AddEventTypeAttributes_APIGatewayProxyRequest_AddsCorrectAttributes_
             Mock.Assert(() => _transaction.SetRequestMethod(inputObject.HttpMethod));
             Mock.Assert(() => _transaction.SetUri(inputObject.Path));
             Mock.Assert(() => _transaction.SetRequestParameters(inputObject.QueryStringParameters));
-            Mock.Assert(() => _transaction.AcceptDistributedTraceHeaders(inputObject.Headers, Arg.IsAny<Func<IDictionary<string, string>, string, IEnumerable<string>>>(), TransportType.HTTP));
 
+            Assert.That(_transportType, Is.EqualTo(TransportType.Unknown));
         });
     }
 
+    [Test]
```

Comment:
Would you consider making this a TestCase so you can test both `https` and `http` values for the `X-Forwarded-Proto` header?

Reply:
Good idea. I'm updating both tests (single and multi header)